src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -1437,10 +1442,23 @@ public WebhookImpl createWebhook(DataObject object)
             }
         }
         
-        return new WebhookImpl(channel, id, type)
+        WebhookImpl webhook = new WebhookImpl(channel, getJDA(), id, type)
                 .setToken(token)
-                .setOwner(owner == null ? null : channel.getGuild().getMember(owner))
+                .setOwner(owner == null || channel == null ? null : channel.getGuild().getMember(owner))
```

Comment:
Is it possible that `owner` wouldn't be null but `channel` would be in this situation and thus this `||` would actually erroneously set `owner` to `null` even though we activelly have it?

Reply:
The problem is we can't get the member instance since there is no guild for it in cache